configure.ac

```diff
@@ -0,0 +1,191 @@
+AC_PREREQ([2.69])
+AC_INIT([rkt], [0.1], [https://github.com/coreos/rkt/issues])
+
+# Sets parameters
+
+# RKT source distribution path
+AC_ARG_WITH(org-path,
+        AS_HELP_STRING([--with-org-path=PATH],
+                [Path to rkt source distribution]),
+        [ORG_PATH="$withval"],
+        [ORG_PATH="github.com/coreos"])
+
+AC_SUBST(ORG_PATH)
+
+# STAGE1 build settings
+
+## STAGE1:type
+AC_ARG_WITH(stage1,
+        AS_HELP_STRING([--with-stage1=type],
+                [type of stage1 build one of 'src', 'coreos', 'none']),
```

Comment:
Please add an information about default value in help string (see above for an example).